common/utils/UuidUtils.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Gerhard
+ * Date: 06-Apr-19
+ * Time: 16:17
+ */
```

Comment:
Need a standard header here. See here for a good example (and only place yourself for the copyright and author):
https://github.com/openemr/openemr/blob/master/library/reminder/src/BirthdayReminder.php#L2-L12